### DISCUSSION TITLE
can I get multiple server's connections from tcp?

### DISCUSSION THREAD

**Discussion by jagonzalezga** [eModbus/eModbus#315] [Q&A]
the esp32 is my client and i have 3 PLCs how can I get three add request in a single program. 

example 

PLC 1 = 192.168.1.20 (SERVER 1)
PLC 2 = 192.168.1.21 (SERVER 2)
PLC 3 = 192.168.1.22 (SERVER 3)
ESP32 = 192.168.1.177 (CLIENT 1)

**Comment by Miq1** [comment]
Have a look at the [documentation](https://emodbus.github.io/modbusclient-tcp-api#bool-settargetipaddress-host-uint16_t-port--uint32_t-timeout--uint32_t-interval) on ``ModbusClientTCP``. 😉
